db/migrate/20191003014319_add_organization_key_to_user_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizationKeyToUserTable < ActiveRecord::Migration[5.2]
```

Comment:
I'm not 100% familiar with the project but I would go with the default Rails way of adding foreign_key here:
`add_reference :users, :organization, foreign_key: true`

This is gonna add `organization_id` column and create `foreign_key` constraint out of the box. Foreign key is going to be helpful to the database for the performance and to maintain the "correct" state(less important here since you have `nulls`).

Also, you can get this with the generator like this:
`bin/rails g migration add_organization_key_to_user_table organization:references`

Reply:
> @LizPrescott yeah in this case referential integrity is less important since there could be `nulls`, but it will still work since you've specified `optional: true` inside the `User` model, and you would create `reference` with `null: true`
> 
> `add_reference :users, :organization, null: true, foreign_key: true`
> 
> But more importantly Rails `add_reference` will create you an index on `oganization_id` column, that could be used for faster lookup in queries such as
> `User.where(organization: @some_organization)` or `Organization.first.users`
> 
> <img alt="Screenshot 2019-10-09 at 09 50 35" width="1276" src="https://user-images.githubusercontent.com/7427365/66461905-453c4c00-ea7a-11e9-9b74-29f377893c7c.png">
> 
> I think that generally it is a good practice to have an index on fk columns.

Turns out this branch is relevant, and I've pushed the new commit with your suggestion. Thanks!